### PR TITLE
IDWAL00K-220: Merge IDWAL00K-220 to 1/14

### PR DESCRIPTION
**Pull request by jvsam**

### CODE REVIEW

**Comment by anri-1151** [comment]
back -ზე რომ აწვები და უკან გაბრუნებს მემგონო სწორ ფორმაზე დავაბრუნე. თუ რამეა ჩავასწორებ უცბად.

File: app/src/templates/guest/welcome.pug [hunk 1]
```diff
@@ -3,7 +3,7 @@
     .content(flex, layout="column", layout-align="center center") 
         .logo
         .header selfkey
-        .subheader blockchain identity wallet
+        .subheader blockchain identity wallet member.setup.choose
```
Comment (and1gio) [inline app/src/templates/guest/welcome.pug]: why there is added **member.setup.choose** at the end of this line ?

Comment (anri-1151) [reply app/src/templates/guest/welcome.pug]: It's nothing. I will delete it. Maybe I copy it and paste it by accident.

**Comment by anri-1151** [comment]
Changes are made only in "app/src/templates/member/setup/choose.pug". Other file changes are made by IDE when I opened them. It is only spaces and etc. nothing else. And it's already pushed and from it these files can't be deleted. So you can merge it. This will not cause problems.

**Pull request merged**